superset/viz.py

```diff
@@ -467,6 +468,36 @@ def get_data(self, df):
     def json_data(self):
         return json.dumps(self.data)
 
+    def fix_df_column_case(self, df):
```

Comment:
This doesn't really belong in `viz.py`, I'd much rather have this in `db_engine_spec.py` and only execute for these weird database. Maybe it's a method in `BaseEngineSpec` that applies conditionally based no a class attribute.

Reply:
Good idea, I'll move it over.